python/py-numpy/Portfile

```diff
@@ -7,7 +7,7 @@ PortGroup           compiler_blacklist_versions 1.0
 
 name                py-numpy
 version             1.26.4
-revision            2
+revision            3
```

Comment:
Does installed files actually differ? If not (and if the port foes not really use `fftw-3`, nothing links to it anyway), no revbump is due.

Reply:
It does need the `revision` increase as it changes `depends_lib`

Reply:
Why, if installation is identical?

Reply:
because the dependency is listed in the registry